feat(fiori-mcp-server): update readme (#3971)
SAP/open-ux-tools: packages/fiori-mcp-server/README.md

- _Please add a SAP Fiori elements list report app to my CAP project_
- _Generate a new CAP project and SAP Fiori app based on my_picture.png_
- _Generate a new SAP Fiori elements list report app for odata service `https://hostname:port/sap/opu/odata4/sap/my_v4_service?sap-client=000`_.

>   ℹ️This prompt requires a Stored System for credentials using extension [Connection Manager for SAP Systems](https://marketplace.visualstudio.com/items?itemName=SAPOSS.sap-ux-sap-systems-ext).

Reviewer: ```suggestion
  ℹ️This prompt requires a saved SAP system connection by using the [Connection Manager for SAP Systems](https://marketplace.visualstudio.com/items?itemName=SAPOSS.sap-ux-sap-systems-ext) extension.
```

- _Add the FCL to the SAP Fiori elements app_

> - _Enable initial load for the fiori app_

Author: Same here. Sample prompt with intended wording. No user will ever write _SAP Fiori elements app_ but the MCP is smart enough to figure out.

The main functionalities are:

- Generating a Fiori elements app within an [SAP Cloud Application Programming Model (CAP)](https://cap.cloud.sap/) project

> - Generating a Fiori elements app within an [ABAP RESTful Application Programming Model (RAP)](https://pages.community.sap.com/topics/abap/rap) project

Reviewer 2: @heimwege Can we add a prompt example around line 14 above?
Something like this
```
- _Generate a new fiori app LROP template for odata service https://hostname:port/sap/opu/odata4/sap/my_v4_service/?sap-client=000_. 

  **Note: Requires Stored System for credentials using extension [Connection Manager for SAP Systems](https://marketplace.visualstudio.com/items?itemName=SAPOSS.** sap-ux-sap-systems-ext) )
```

Author: Added in e5bbefc38f11c62c694f66bbaf2c85877fe8963e

The main functionalities are:

- Generating an SAP Fiori elements app within an [SAP Cloud Application Programming Model (CAP)](https://cap.cloud.sap/) project

> - Generating an SAP Fiori elements app for an existing OData service, e.g. an [ABAP RESTful Application Programming Model (RAP)](https://pages.community.sap.com/topics/abap/rap) based OData service

Reviewer: ```suggestion
- Generating an SAP Fiori elements app for an existing OData service, for example, an [ABAP RESTful Application Programming Model](https://pages.community.sap.com/topics/abap/rap) based OData service
```

Author: fixed in 01835fb171ea85d51e6a5ff2d4d456279ea9705b

- Adding and deleting pages from an app
- Adding and modifying controller extensions

> - Modifying `manifest.json` properties depending on the app (e.g. adding Flexible Column Layout, enabling initial load)

Reviewer: ```suggestion
- Modifying `manifest.json` properties depending on the app, for example, adding a flexible column layout or enabling initial load
```

#### `execute_functionality` (Step 3 of 3)
Executes a specific functionality to create a new or modify an existing SAP Fiori application with provided parameters.

## [Logging](#logging)

> By default, logging is done for level `error`. To adjust the log level or disable logging completely you can use the `env` property in your configuration.

Reviewer: ```suggestion
By default, logging is enabled at the `error `level `. To adjust the log level or disable logging completely, you can use the `env` property in your configuration.
```

```json
{
  "mcpServers": {
    "fiori-mcp": {
      "type": "stdio",
      "timeout": 600,
      "command": "npx",
      "args": ["--yes","@sap-ux/fiori-mcp-server@latest", "fiori-mcp"],
      "env": {
        "LOG_LEVEL": "debug"
      }
    }
  }
}
```

> The following log levels are supported: `off`, `error`, `warn`, `info`, `debug`, `verbose`.

Reviewer: ```suggestion
The following log levels are supported: `off`, `error`, `warn`, `info`, `debug`, and `verbose`.
```

Author: fixed in 01835fb171ea85d51e6a5ff2d4d456279ea9705b
